recipes/fast-query-parsers/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "fast-query-parsers" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fast_query_parsers-{{ version }}.tar.gz
+  sha256: df972c0b58d0bf51fa43b67d2604ab795984015d47552d02175ebcc685e4852b
+
+build:
+  skip: True  # [py<38]
```

Comment:
```suggestion
  skip: True  # [py<38] or [win]
```

Reply:
My bad I was doing another  recipe that was noarch and got confused